### PR TITLE
[zh] Update debug-link

### PR DESCRIPTION
**Pull request by my-git9**
[zh] Update debug-link

### CODE REVIEW

File: content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md [hunk 1]
```diff
@@ -95,15 +95,15 @@ This may be caused by a number of problems. The most common are:
...
+[Debugging Kubernetes nodes with crictl](/docs/tasks/debug/debug-cluster/crictl/).
 -->
 这可能是由许多问题引起的。最常见的是：
 
 - 网络连接问题。在继续之前，请检查你的计算机是否具有全部联通的网络连接。
 - 容器运行时的 cgroup 驱动不同于 kubelet 使用的 cgroup 驱动。要了解如何正确配置 cgroup 驱动，
   请参阅[配置 cgroup 驱动](/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/)。
 - 控制平面上的 Docker 容器持续进入崩溃状态或（因其他原因）挂起。你可以运行 `docker ps` 命令来检查以及 `docker logs` 命令来检视每个容器的运行日志。
-  对于其他容器运行时，请参阅[使用 crictl 对 Kubernetes 节点进行调试](/zh/docs/tasks/debug-application-cluster/crictl/)。
+  对于其他容器运行时，请参阅[使用 crictl 对 Kubernetes 节点进行调试](/zh/docs/tasks/debug/debug-cluster/crictl/)。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:106]: we need to move the crictl page first?

Comment (my-git9) [reply content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:106]: I found that there is some PRs to do similar things, we can wait for it to finish before doing this merge ?

Comment (tengqm) [reply content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:106]: yes I  think so.

Comment (my-git9) [reply content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:106]: crictl page  has been moved, please review again, thanks

**Pull request merged**